Cargo.lock

```diff
@@ -0,0 +1,604 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Usually we don't include Cargo.lock file in the repo.

Reply:
It's removed now.